docker/docker-compose.yml

```
@@ -47,6 +47,9 @@ services:
        WORDPRESS_DB_HOST: wordpress-db:3306
        WORDPRESS_DB_USER: wordpress
        WORDPRESS_DB_PASSWORD: wordpress
+       # These values can be overridden using .env file
```

Thought it might be clearer if we make it explicit, especially if we add anything else here in the future:
```suggestion
       # YOTI_CONNECT_API and YOTI_CONNECT_BASE_URL can be overridden using the .env file
```